src/drivers/serac.cpp

```diff
@@ -34,7 +36,8 @@ int main(int argc, char *argv[])
   MPI_Comm_rank(MPI_COMM_WORLD, &myid);
 
   // mesh
-  const char *mesh_file = "../../data/beam-hex.mesh";
+  std::string base_mesh_file = std::string(SERAC_SRC_DIR) + "/data/beam-hex.mesh";
```

Comment:
I assume `SERAC_SRC_DIR` is in the config header. Should we change the tests do it this way instead?

Reply:
Would you like me to remove the command line path? I could push that to this PR if you want.  It is one less thing to remember / copy&paste.

Reply:
I wouldn't worry about it now. I can add it to my next PR.